### ISSUE TITLE
сколько режимов существует?

### ISSUE THREAD

**Issue by zetlnd** [Rast1234/VOffline#2]
какие режимы можно задать в строках
    "Targets": [ "club167809706" ],
    "Modes": [ "wall" ],
photos

**Comment by zetlnd** [comment]
"Discussion" или иначе надо вводить?

**Comment by Rast1234** [comment]
согласен, много не допилено, но руки не дошли. video не работает даже. в основном сам пользовался wall и photos, они более менее все вытягивают.
